doc/howto/move_instances.md

```diff
@@ -63,6 +63,13 @@ To do so, ensure the following configuration:
 If you are using a shared storage pool to back your instance like Ceph RBD, you don't need to set the [`size.state`](devices-disk) property.
 ```
 
+The default value for {config:option}`instance-migration:migration.stateful` was introduced with the {config:option}`server-miscellaneous:instances.migration.stateful` introduced with the `stateful_migration_default` API extension and applies only to new instances. Existing instances won't be affected by this change.
+
+```{note}
+Because LXD disables virtiofs file system shares when {config:option}`instance-migration:migration.stateful` is enabled, and only exports them via 9p protocol, some guest OSes (like Centos 8) that do not support 9P file system share, won't be able to have a shared file system with the host (unless you disable stateful migration).
```

Comment:
Can you try and make this a bit more concise without mentioning file system so often.